packages/paste-website/src/pages/components/index.mdx

```diff
@@ -19,18 +20,38 @@ import {SidebarCategoryRoutes} from '../../constants';
   Components
 </Heading>
 
-The components listed here are on our roadmap.  Only a few of them are available today.
+The components listed here are on our roadmap. Only a few of them are available today.
 
 If you’re planning on using an alpha or beta component, please check in with the Paste team through the <strong>#help-design-system</strong> channel on Slack.
 
-Don't see a component you need listed here? [Create an issue on GitHub](https://github.com/twilio-labs/paste/issues), and we'll evaluate it. Not all component
-requests are accepted for the design system. We’ll have guidelines soon for what to do if your component isn’t
-supported.
+<Callout>
+  <CalloutTitle as="h4">Don't see a component you need listed here?</CalloutTitle>
+  <CalloutText>
+    We've got a few options for you.
+    <ul>
+      <li>
```

Comment:
There are `<UL>` `<LI>` components that you can use too instead of plain html elements.

```
import {UL, LI} from '../../components/typography';

...

<UL>
  <LI>
    etc...
```

Reply:
In fact you need to pull those out of `CalloutText` entirely.

```
<Callout>
  <CalloutTitle as="h4">Don't see a component you need listed here?</CalloutTitle>
  <CalloutText> ... </CalloutText>
  <UL>
    <LI>
...
  </UL>
</Callout>
```